src/content/docs/tutorial-service-level-mgmt/establish-capability-sli.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Establish capability SLIs"
```

Comment:
This doc is a little confusing. I remember having Andrew demo this for me a while back for the 'peak performance' tutorial, so I have a rough idea of why you'd want to do this. Still, the "goal" of this from the intro ("service lvls on important capabilities within your system") doesn't really "Stick" with the steps you've outlined here. 

- how is this service level different from other service levels? why wouldn't you *only* put SLs on important capabilities? 
- I wonder if these 10 steps can be chunked into 3-4 "broad" steps with subtasks within them. For example, steps 1-2 could be merged and titled "Find your baselines for your key capabilities" [do these tasks with NRQL]; steps 3-4 don't have a clear "Goal" like steps 1-2 do -- you mentioned that we'd need it in a moment but what do we do with this information in step 5? What is the "thing" we're meant to create that's different from the other tutorials?

Reply:
https://docs.newrelic.com/docs/tutorial-peak-demand/find-your-baseline/ i have a somewhat similar doc here but i remember struggling with this. i think if you read thru this, you can take wwhat you think doesnt work and maybe apply what should've been done better in your tutorial. I think btwn the two of them there might be a clearer outcome

Reply:
fwiw i think this tutorial was extremely not good but sometimes it helps to look at something worse to make a better iteration of the thing